spec/unit_spec_helper.rb

```diff
@@ -55,6 +56,7 @@
   UnitTests::ColumnTypeHelpers.configure_example_group(config)
 
   config.include UnitTests::Matchers
+  config.include RSpec::Matchers::FailMatchers
```

Comment:
I didn't know about this [matchers](http://www.rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers/FailMatchers) :star: 

I think we can drop our [custom](https://github.com/thoughtbot/shoulda-matchers/blob/master/spec/support/unit/matchers/fail_with_message_matcher.rb) [matchers](https://github.com/thoughtbot/shoulda-matchers/blob/master/spec/support/unit/matchers/fail_with_message_including_matcher.rb) and use this provided by RSpec at some point.